installer/IL15_emulated_checks_init.sh

```diff
@@ -111,12 +111,10 @@ IL15_emulated_checks_init() {
         sed -i 's/routersploit\.log/\/tmp\/routersploit\.log/' ./rsf.py
       fi
       # patch some code to write the history files to a subdirectory in /root
-      # shellcheck disable=SC2088
-      if grep -q  '~/.history' ./routersploit/interpreter.py; then
+      if grep -q  ~/'.history' ./routersploit/interpreter.py; then
         sed -i 's/~\/\.history/~\/\.routersploit\/\.history/' ./routersploit/interpreter.py
       fi
-      # shellcheck disable=SC2088
-      if grep -q  '~/.rsf_history' ./routersploit/interpreter.py; then
+      if grep -q  ~/'.rsf_history' ./routersploit/interpreter.py; then
```

Comment:
hmmm ... but, now the `~` expands. And we do not want to expand it. We really would like to grep for a tilde and not for /home/something or /root

Reply:
> hmmm ... but, now the `~` expands. And we do not want to expand it. We really would like to grep for a tilde and not for /home/something or /root

The other option would be to use $HOME according to the wiki: https://www.shellcheck.net/wiki/SC2088